plugins/modules/wafv2_web_acl.py

```diff
@@ -338,6 +417,10 @@ def main():
     purge_rules = module.params.get("purge_rules")
     check_mode = module.check_mode
 
+    custom_response_bodies = {}
+    for name, body in module.params.get("custom_response_bodies").items():
+        custom_response_bodies[name] = snake_dict_to_camel_dict(body, capitalize_first=True)
```

Comment:
By adding the check here you have access to module.

Reply:
Yes, I know that's the better option, was just not putting it there as I already set a default of empty dict for response bodies, but I guess by checking the `len` of dict (as I now do anyway) it could be easily moved here.

Reply:
No need to check the length, an empty dict evaluates to False
```
$ python
Python 3.6.8 (default, Mar 18 2021, 08:58:41) 
[GCC 8.4.1 20200928 (Red Hat 8.4.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> if {}:
...     True
... else:
...     False
... 
False